.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,13 +4,43 @@ There are a few simple things to check before submitting your pull request
 that can help with the review process. You should delete these items
 from your submission, but they are here to help bring them to your attention.
 -->
+# Pull Request
 
-## Issues
-<!-- Link to related issues. Use closing keywords when appropriate -->
+*Issue link(s)*:
 
-## Summary
+<!--
+  Add Related Issues / PRs for context. Eg:
+    Related to elastic/repo#999
+    Resolves #123
+  If there is no issue link, take extra care to write a clear summary and label the PR just as you would label an issue to give additional context to reviewers.
+-->
+
+## Summary - What I changed
+
+<!--
+  Summarize your PR. Animated gifs are 💯. Code snippets are ⚡️. Examples & screenshots are 🔥
+-->
+
+## How To Test
+
+<!--
+  Some examples of what you could include here are:
+  * Links to Buildkite results for CI test improvements
+  * Telemetry before/after screenshots (or short videos showing how something works)
+  * Copy/pasted commands and output from the testing you did in your local terminal window
+  * If tests run in Buildkite, you can 🪁or 🔱, respectively, to indicate tests will run in CI
```

Comment:
is this applicable? afaik, Buildkite use is limited to the automatic React trigger action